frontend/src/pages/Privacy/PrivacyListItem/PrivacyListItem.module.scss

```diff
@@ -0,0 +1,89 @@
+@import '/src/variables';
+@import '/src/mixins';
+
+.itemTitle {
+  margin-bottom: 26px;
+  font-size: 26px;
+  line-height: 34px;
+  font-weight: 500;
+
+  &_number {
+    color: $page-accent-color;
+  }
+}
+
+.description {
+  margin-bottom: 20px;
+  font-size: 22px;
+  line-height: 34px;
+}
+
+.subList {
```

Comment:
As for me, the bullet and text are not visually centered

Reply:
Yes, it looks like that. But we were discussing the height of the line long ago and how it looks in the browser. Have you been able to find any possible solution for this?

Reply:
Ok, sorry, was not aware you have discussed it already) I guess top:45 is not an option here?